daml-lf/spec/daml-lf-1.rst

```diff
@@ -2138,6 +2254,18 @@ types that satisfies the following rules::
   ——————————————————————————————————————————————————— TypeOrderTypeAppRight
     τ σ₁ <ₜ τ σ₂
 
+  ——————————————————————————————————————————————————— TypeOrderTyAppAnyException
+    τ σ <ₜ 'AnyException'
+
+  ——————————————————————————————————————————————————— TypeOrderAnyExceptionGeneralError
+    'AnyException' <ₜ 'GeneralError'
+
+  ——————————————————————————————————————————————————— TypeOrderGeneralErrorArithmeticError
+    'GeneralError' <ₜ 'ArithmeticError'
+
+  ——————————————————————————————————————————————————— TypeOrderArithmeticErrorContractError
+    'ArithmeticError' <ₜ 'ContractError'
+
```

Comment:
Those are primitive type, they should be bigger than any non primitive type. 
I think the good place is between `Update` and `Mod:T`  line 2092

Reply:
That makes sense.